docs/USAGE-GUIDE.md

```diff
@@ -23,7 +23,7 @@ are using CMake that step is unnecessary. Just follow the instructions here to u
 
 (Required): You need at least CMake version 3.0 to fully benefit from Modern CMake. See [this](https://www.youtube.com/watch?v=bsXLMQ6WgIk) for more information.
 
-(Optional): Set the CMake variable `CMAKE_INSTALL_PREFIX` to the location libcrypto is installed to. If you do not,
+(Optional): Set the CMake variable `DCMAKE_PREFIX_PATH` to the location libcrypto is installed to. If you do not,
```

Comment:
The `-D` is shorthand for define, so probably shouldn't be included

```suggestion
(Optional): Set the CMake variable `CMAKE_PREFIX_PATH` to the location libcrypto is installed to. If you do not,
```

Reply:
or replace variable with flag and make it `-DCMAKE_PREFIX_PATH`